README.md

```diff
@@ -114,6 +114,41 @@ In the other OSs not listed above, the pure-java SQLite is used. (Applies to ver
 
 If you want to use the native library for your OS, [build the source from scratch](./CONTRIBUTING.md).
 
+## GraalVM native-image support
+
+Sqlite JDBC supports [GraalVM native-image](https://www.graalvm.org/native-image/) out of the box starting from version 3.40.1.0.
+There has been rudimentary support for some versions before that, but this was not actively tested by the CI.
+
+By default, the `sqlitejdbc` library for the compilation target will be included in the native image, accompanied the required JNI configuration.
+At runtime, this library will be extracted to the temp folder and be loaded from there.
+For the best startup performance however, it is recommended to set the `org.sqlite.lib.exportPath` property at build-time.
+This will export the `sqlitejdbc` library at build-time to the specified directory, after which the library will no longer be included as a resource.
```

Comment:
```suggestion
This will export the `sqlitejdbc` library at build-time to the specified directory, and the library will not be included as a resource.
```